wrappers/csharp/cs-tutorial-8-D400-on-chip-calibration/Window.xaml.cs

```diff
@@ -57,6 +57,12 @@ public CaptureWindow()
                 using (var ctx = new Context())
                 {
                     var devices = ctx.QueryDevices();
+                    if (0==devices.Count)
+                    {
+                        Console.WriteLine("The tutorial {0} requires Realsense D400 device to run.\nConnect a device and rerun",

```

Comment:
QueryDevices doesn't ask for a D400... if you have L500, then you'll get a device...

Reply:
Done